beacon-chain/blockchain/process_block.go

```diff
@@ -693,37 +614,25 @@ func (s *Service) areDataColumnsAvailable(ctx context.Context, root [32]byte, si
 		return errors.Wrap(err, "peer info")
 	}
 
-	// Exit early if the node is not expected to custody any data columns.
-	if len(peerInfo.CustodyColumns) == 0 {
-		return nil
-	}
-
 	// Subscribe to newsly data columns stored in the database.
```

Comment:
Random typo spotted `newsly`

Reply:
Fixed in 7fd169da5f436c1fe9fabe40a8396576b13d3157.